static/app/views/dashboards/widgets/timeSeriesWidget/timeSeriesWidgetVisualization.tsx

```diff
@@ -279,13 +328,83 @@ export function TimeSeriesWidgetVisualization(props: TimeSeriesWidgetVisualizati
     })(deDupedParams, asyncTicket);
   };
 
+  const yAxes: YAXisComponentOption[] = [leftYAxis, rightYAxis].filter(axis => !!axis);
+
   let visibleSeriesCount = props.plottables.length;
   if (releaseSeries) {
     visibleSeriesCount += 1;
   }
 
   const showLegend = visibleSeriesCount > 1;
 
+  // Keep track of which `Series[]` indexes correspond to which `Plottable` so
+  // we can look up the types in the tooltip. We need this so we can find the
+  // plottable responsible for a given value in the tooltip formatter. The only
+  // tool ECharts gives us is the `seriesIndex` properly. Any given `Plottable`
+  // can be mapped to 1 or more `Series`, so we need to maintain a reverse
+  // lookup
```

Comment:
Question: When can a `Plottable` be mapped to more than one series?

Reply:
A few cases:

- if the plottable has an ingestion delay, it becomes two series
- if the plottable is a set of span samples, it's one series per sample (coming soon)
- if the plottable is thresholds lines, there are three series (coming soon)

etc.